wp-content/mu-plugins/stat-collector/StatCollectorFunctions.php

```diff
@@ -38,9 +32,6 @@ function drools_response($response, $uid) {
 
 function results_sent($type, $to, $uid, $url = null, $message = null) {
   $db = _get_db();
-  $db->suppress_errors();
-  $db->show_errors();
-
   $result = $db->query($db->prepare("
    INSERT into messages (uid, msg_type, address, url, message) VALUES (%s, %s)",
     $uid,
```

Comment:
The `error_log` needs to be updated because `data` doesn't exists in the arguments sent to the function. You need to include the `uid`, `msg_type`, `address`, `url`, and `message`.